README.md

```diff
@@ -156,19 +156,54 @@ Please see [Getting Started](./docs/getting-started.md) for an introduction to t
 
 ## Configuration
 
-Please see the documentation below for details about the ORT configuration.
-
-* The [ORT configuration](./model/src/main/resources/reference.conf) file - the main configuration file for the
-  operation of ORT. This configuration is maintained by an administrator who manages the ORT instance. In contrast to
-  the configuration files in the following, this file rarely changes once ORT is operational.
-* The [.ort.yml](./docs/config-file-ort-yml.md) file - project-specific license finding curations, exclusions
-  and resolutions to address issues found within a project's code repository.
-* The [package configuration](./docs/config-file-package-configuration-yml.md) file - package (dependency) and provenance
-  specific license finding curations and exclusions to address issues found within a scan result for a package.
-* The [curations.yml](./docs/config-file-curations-yml.md) file - curations correct invalid or missing package metadata
-  and set the concluded license for packages.
-* The [resolutions.yml](./docs/config-file-resolutions-yml.md) file - resolutions allow *resolving* any issues
-  or policy rule violations by providing a reason why they are acceptable and can be ignored.
+The following provides an overview over the various configuration files that can be used to customize ORT behavior.
```

Comment:
"overview of" sounds better than "overview over".

Commit message: "customize", "rules.kts"

Reply:
Done.